gpii/node_modules/flowManager/src/UntrustedFlowManager.js

```diff
@@ -68,15 +68,22 @@ fluid.defaults("gpii.flowManager.untrusted.stateChangeHandler", {
                 "{flowManager}.settingsDataSource",
                 "{that}.userToken",
                 "{arguments}.0", // device reporter data
-                "{that}.events.onSettings",
+                "{that}.events.onMatchDone",
                 "{that}.events.onError"
             ]
+        },
+        addLifecycleInstructions: {
+            funcName: "gpii.flowManager.addLifecycleInstructions",
+            args: ["{flowManager}", "{arguments}.0", "{arguments}.1"]
+                                    // full payload, success event
         }
     },
     events: {
         onUserToken: null,
         onDeviceContext: null,
-        onSettings: null
+        onContextAdded: null,
+        onLifecycleInstructions: null,
```

Comment:
Better for this to be just one event with prioritised listeners as in FlowManagerRequests rather than a bunch of events with different names